medical/gql_mutations.py

```diff
@@ -110,13 +103,17 @@ def reset_item_or_service_before_update(item_service):
         "category", # service only
         "package",  # item only
         "quantity", # item only
+        "packagetype", #service only
+        "manualPrice", #service only
     ]
     for field in fields:
         if hasattr(item_service, field):
             setattr(item_service, field, None)
 
 
 def update_or_create_item_or_service(data, user, item_service_model):
+    items = data.pop('items') if 'items' in data else []
```

Comment:
I need to check here, for the product this code lead to removal of all item/services if the item and services were not loaded.
I aim to fix  the product by having the default as None and in that case I could assume that no change was requested, this approach have the limitation that the FE should send the section items/services only when loaded beforehand; (FE part not implemented in Product therefore I consider [] as no change)

Reply:
this is not blocking for the merge

Reply:
Fixed